data/tutorials/gs_00_up_and_running.md

```diff
@@ -11,88 +11,166 @@ date: 2021-05-27T21:07:30-00:00
+# Archlinux
+pacman -S opam
 
+# Debian (stable, testing and unstable)
+apt-get install opam
 ```
-# environment setup
-$ opam init
-$ eval `opam env`
 
-# install a specific version of the OCaml base compiler
-$ opam switch create 4.14.0
-$ eval `opam env`
+**Binary distribution**
 
-# install dev tools, hit Enter to confirm at Y/n prompt
-$ opam install dune utop ocaml-lsp-server
+``` shell
+bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
 ```
 
-After the `opam init` command, you might get a result asking if you'd like to update your `zsh` configuration. If you get that message, type in `N`, the default, then type `y` to install the hook, which will run `eval $(opam env)`. As you get more well-versed in OCaml, you can change these settings by rerunning `opam init`.
+### Initialising `opam`
 
-**Please note**: The back ticks shown around `opam env` after `eval` are essential. They change the order of application, which is very important. The back ticks tells the system to first evaluate `opam env` (which returns a string of commands) and then `eval` executes those commands in the string. Executing them doesn't return anything, but it initializes the Opam environment behind the scenes.
+This step is done automatically by the alpha-version platform installer, [see below](#up-and-running-with-the-platform-installer).
 
-Check that the installation was successful by running `opam --version`. Please note merely using `opam init` might install a previous version of Opam. The most current version can be found at [opam.ocaml.org](https://opam.ocaml.org/packages/ocaml-base-compiler/).
+If you have installed the binary distribution of `opam` through the install script, this step should already be done. If you have installed it through your system package manager, you must initialise `opam` by running the following command:
 
-The OCaml base compiler installation uses the `opam switch create` command; `switch` is used to have several installations on disk, like packages, compiler version, etc. Specify which version at the end as shown above, i.e., 4.14.0. All possible compiler versions can be found with `opam switch list-available`.
+```
+opam init          # Can take some time
+eval $(opam env)
+```
 
-Next, check that OCaml is installed properly with the following commands. The line beneath the $ command shows the desired output for both the OCaml version and the toplevel version (installed specifically with the above `switch` command):
+The first command (`opam init`) creates a first switch, usually called `default`, although this is just a convention. If you have installed `ocaml` through your system package manager, the first switch will be set up to use this compiler (it is called a "system switch"). Otherwise, it will build one from source, usually taking the most recent version of `ocaml`.
```

Comment:
```suggestion
The first command (`opam init`) creates a first switch, usually called `default`, although this is just a convention. If you have installed OCaml through your system package manager, the first switch will be set up to use this compiler (it is called a "system switch"). Otherwise, it will build one from source, usually taking the most recent version of OCaml.
```